src/src/Controller/OPController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class OPController extends AbstractController
+{
+    #[Route('/o/p', name: 'app_o_p')]
```

Comment:
Names (variables, functions, classes, ...) should ideally be self-explainable. So instead of `app_o_p`, I would suggest something like `app_terms` or `app_terms_of_service`, etc. The same goes for paths, so instead of `/o/p`, I would suggest `/obchodni-podminky`.